cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -79,6 +80,7 @@ class AnalyzerCommand : CliktCommand(name = "analyze", help = "Determine depende
     ).convert { it.expandTilde() }
         .file(mustExist = true, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = true)
         .convert { it.absoluteFile.normalize() }
+        .default(ortConfigDirectory.resolve(ORT_CURATIONS_FILENAME))
```

Comment:
As discussed offline, I have changed it to handle the curations file option like the other configuration file optoins to replace the global file instead of amending it.

Reply:
BTW, because of

> Note that the `--package-curation-file` option of the evaluator does not have a default location, because it would overwrite the curations used by the analyzer which could lead to unexpected results.

I'd actually vote for renaming `--package-curation-file` in the evaluator to something like `--override-package-curation-file`, to denote this is not a usual configuration option that is commonly used, but rather for testing new curations without going through the analyze / scan cycle again.

Reply:
And similar for other "override-for-testing"-style options.

Reply:
I agree, I'll take a note to change that in a separate PR, because it requires changes in our internal scripts.

Reply:
Thanks!